x-pack/plugins/canvas/public/lib/element_handler_creators.ts

```diff
@@ -71,6 +70,7 @@ export const basicHandlerCreators = {
     image = ''
   ): void => {
     const notifyService = pluginServices.getServices().notify;
+    const customElementService = pluginServices.getServices().customElement;
```

Comment:
Sidebar: All of these "element handlers" would make good candidates for hooks that could just be added to components where needed.